srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/PopulatedDisplayGroup.java

```diff
@@ -0,0 +1,204 @@
+            }
+            else if(var instanceof MultiSelectVariable)
+            {
+                varView = new MultiSelectVariableView((MultiSelectVariable) var, "");
+            }
+            else if(var instanceof ProcedureVariable)
+            {
+                varView = new ProcedureVariableView((ProcedureVariable) var, "");
+            }
+            else
+            {
+                LOGGER.debug("Unexpected Variable type while creating a PopulatedDisplayGroup. Key: {}",
+                        var.getKey());
+                // Unexpected variable type
+                continue;
+            }
+            fDisplayItems.add(varView);
+        }
+        
+        // These group names are all well-known and are hard-coded as such.
+        if(group.getName().equalsIgnoreCase("Clinical Conditions or Diseases - Recent"))
+        {
+            final List<String> factorList = new ArrayList<String>();
+            // Output the date without the time of day.
+            final SimpleDateFormat format = new SimpleDateFormat("MM/dd/yy");
+            for(final HealthFactor factor: patient.getHealthFactors())
+            {
+                factorList.add(String.format("%s %s%n", format.format(factor.getDate()), factor.getFactor()));
+            }
+            final DisplayItem refInfo = new ReferenceItem("Health Factors", variables.get(0).getGroup(), factorList);
+            fDisplayItems.add(0, refInfo);
+        }
+    }
+
+    /**
+     * <p>Returns the List of Variables, always containing at least 1 Variable. All
+     * contained Variables will have equal {@link VariableGroup}s. The
+     * collection itself is immutable, though the variables themselves are not.</p>
+     * 
+     * <p>Order is preserved from construction.</p>
+     */
+    public List<Variable> getVariables()
+    {
+        return Collections.unmodifiableList(fVariables);
+    }
+    
+    /**
+     * <p>Returns the List of DisplayItems, always containing at least 1 VariableView. All
+     * contained VariableViews will have equal {@link VariableGroup}s. The
+     * collection itself is immutable, as well as the VariableViews</p>
+     * 
+     * <p>Order is preserved from construction.</p>
+     */
+    public List<DisplayItem> getDisplayItems()
+    {
+        return Collections.unmodifiableList(fDisplayItems);
+    }
+    
+    public VariableGroup getGroup()
+    {
+        // Since all Variables have the same group, just use the first's group.
+        return fDisplayItems.get(0).getDisplayGroup();
+    }
+    
+    /**
+     * Returns the VariableGroup's name.
+     */
+    public String getName()
+    {
+        return getGroup().getName();
+    }
+    
+    @Override
+    public boolean equals(Object obj)
+    {
+        if (obj instanceof PopulatedDisplayGroup)
+        {
+            final PopulatedDisplayGroup other = (PopulatedDisplayGroup)obj;
+            return this.getVariables().equals(other.getVariables());
```

Comment:
Why does `equals()` depend only on the variables? (I have a guess but it should be documented.)